test_scripts/Smoke/API/012_PerfomInteraction_PositiveCase.lua

```diff
@@ -0,0 +1,428 @@
+								{ hmiLevel = level, audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" },
+								{ hmiLevel = level, audioStreamingState = "NOT_AUDIBLE", systemContext = "VRSESSION" },
+								{ hmiLevel = level, audioStreamingState = "ATTENUATED", systemContext = "VRSESSION" },
+								{ hmiLevel = level, audioStreamingState = "ATTENUATED", systemContext = "HMI_OBSCURED" },
+								{ hmiLevel = level, audioStreamingState = "AUDIBLE", systemContext = "HMI_OBSCURED" },
+								{ hmiLevel = level, audioStreamingState = "AUDIBLE", systemContext = "MAIN" })
+							:Times(6)
+					elseif request == "VR" then
+						self.mobileSession1:ExpectNotification("OnHMIStatus",
+								{ systemContext = "MAIN", hmiLevel = level, audioStreamingState = "ATTENUATED" },
+								{ systemContext = "MAIN", hmiLevel = level, audioStreamingState = "NOT_AUDIBLE" },
+								{ systemContext = "VRSESSION", hmiLevel = level, audioStreamingState = "NOT_AUDIBLE" },
+								{ systemContext = "VRSESSION", hmiLevel = level, audioStreamingState = "AUDIBLE" },
+								{ systemContext = "MAIN", hmiLevel = level, audioStreamingState = "AUDIBLE" })
+							:Times(5)
+						    :Timeout(timeout)
+					elseif request == "MANUAL" then
+						self.mobileSession1:ExpectNotification("OnHMIStatus",
+								{ systemContext = "MAIN", hmiLevel = level, audioStreamingState = "ATTENUATED" },
+								{ systemContext = "HMI_OBSCURED", hmiLevel = level, audioStreamingState = "ATTENUATED" },
+								{ systemContext = "HMI_OBSCURED", hmiLevel = level, audioStreamingState = "AUDIBLE" },
+								{ systemContext = "MAIN", hmiLevel = level, audioStreamingState = "AUDIBLE" })
+							:Times(4)
+						    :Timeout(timeout)
+					end
+			elseif
+				self.isMediaApplication == false then
+					if request == "BOTH" then
+						self.mobileSession1:ExpectNotification("OnHMIStatus",
+								{ hmiLevel = level, audioStreamingState = "NOT_AUDIBLE", systemContext = "VRSESSION" },
+								{ hmiLevel = level, audioStreamingState = "NOT_AUDIBLE", systemContext = "HMI_OBSCURED" },
+								{ hmiLevel = level, audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+							:Times(3)
+						    :Timeout(timeout)
+					elseif request == "VR" then
+						self.mobileSession1:ExpectNotification("OnHMIStatus",
+								{ systemContext = "VRSESSION", hmiLevel = level, audioStreamingState = "NOT_AUDIBLE" },
+								{ systemContext = "MAIN", hmiLevel = level, audioStreamingState = "NOT_AUDIBLE" })
+							:Times(2)
+						    :Timeout(timeout)
+					elseif request == "MANUAL" then
+						self.mobileSession1:ExpectNotification("OnHMIStatus",
+								{ hmiLevel = level, audioStreamingState = "NOT_AUDIBLE", systemContext = "HMI_OBSCURED" },
+								{ hmiLevel = level, audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+							:Times(2)
+					end
+			end
+	elseif
+		level == "LIMITED" then
+			if
+				self.isMediaApplication == true or
+				commonSmoke.HMITypeStatus.Navigation == true then
```

Comment:
@GetmanetsIrina In common module 'HMITypeStatus' table has 2 keys: NAVIGATION and COMMUNICATION.
So exact (uppercase) key values have to be used when using this table.

Reply:
@dboltovskyi, updated.